entry_types/scrolled/package/src/frontend/MediaPlayer/PlayerContainer.js

```diff
@@ -23,7 +23,7 @@ function PlayerContainer({
         playsInline: playsInline,
         loop: loop,
         controls: controls,
-        hooks: atmo.createMediaPlayerHooks(atmoDuringPlayback)
+        hooks: atmoDuringPlayback ? atmo.createMediaPlayerHooks(atmoDuringPlayback) : {}
```

Comment:
As discussed, depending on user expectation, this could be right or wrong. Let's find out what happens in case of simultaneous usage of background video and atmo audio in the case where the video has sound. It might also be interesting to see whether behaviors differ between paged and scrolled, because in this case it might be an oversight in scrolled

Reply:
In pageflow-paged background video and atmo play simultaneously.
And here in scrolled as well.